### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by mwellick**

### CODE REVIEW

**Comment by mwellick** [comment]
Not good, I found some mistakes

**Comment by mwellick** [comment]
My code doesn't work correctly
Found some mistakes
I'am preparing new solution

**Comment by mwellick** [comment]
I hope this solution will be better :)

File: app/create_file.py [hunk 1]
```diff
@@ -1 +1,45 @@
-# write your code here
+import os
+import sys
+from datetime import datetime
+from typing import Any
+
+
+def create_file_with_content(file_path: str) -> Any:
```
Comment (EdAlekseiev) [inline app/create_file.py:7]:
```suggestion
def create_file_with_content(file_path: str) -> None:
```

File: app/create_file.py [hunk 2]
```diff
@@ -1 +1,45 @@
...
+        while True:
+            input_line = input("Enter content line:")
+            if input_line == "stop":
+                file.write(" \n")
+                break
+            file.write(f"{number_of_content} {input_line}\n")
+            number_of_content += 1
+
+
+def main() -> Any:
```
Comment (EdAlekseiev) [inline app/create_file.py:23]:
```suggestion
def main() -> None:
```

File: app/create_file.py [hunk 3]
```diff
@@ -1 +1,45 @@
-# write your code here
+import os
+import sys
+from datetime import datetime
+from typing import Any
+
+
+def create_file_with_content(file_path: str) -> Any:
+    current_time = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+    content_w_time = f"{current_time}"
```
Comment (EdAlekseiev) [inline app/create_file.py:9]: You don't need use f-string here because `current_time` is already string

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]
Good job

File: app/create_file.py (hunk 3 above)
Comment (mwellick) [reply app/create_file.py:9]: i forgot to change after my previous solution

**Review by EdAlekseiev — APPROVED** [review]
Good job

**Pull request closed without merging**